<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="listWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>799</width>
       <height>509</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonGenerar">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>540</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Archive">
    <property name="title">
     <string>[&amp;Archive]</string>
    </property>
    <addaction name="actionRead_Template"/>
   </widget>
   <addaction name="menu_Archive"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRead_Template">
   <property name="text">
    <string>Read Template</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
